[Bug]: Test case test_clone_when_full failed with TypeError

### Description
The test case tests/functional/pv/pvc_clone/test_clone_when_pvc_full.py::TestCloneWhenFull::test_clone_when_full failed with the error given below

```
self = <tests.functional.pv.pvc_clone.test_clone_when_pvc_full.TestCloneWhenFull object at 0x7ff54724cdc0>
pvc_clone_factory = <function pvc_clone_factory_fixture..factory at 0x7ff52907b670>
pod_factory = <function pod_factory_fixture..factory at 0x7ff526921940>
threading_lock = <unlocked _thread.RLock object owner=0 count=0 at 0x7ff531b76420>


def test_clone_when_full(self, pvc_clone_factory, pod_factory, threading_lock):
    """
    Create a clone from an existing PVC when the PVC is 100% utilized.
    Verify data integrity.
    Verify utilization alert in cloned PVC.
    Expand cloned PVC and ensure utilization alerts are stopped.

    """
    pvc_size_expanded = 6
    file_name = "fio_full"
    prometheus_api = PrometheusAPI(threading_lock=threading_lock)

    # Run IO to utilize 100% of volume
    log.info("Run IO on all pods to utilise 100% of PVCs")
    for pod_obj in self.pods:
        # Get available free space in M
        df_avail_size = pod_obj.exec_cmd_on_pod(
            command=f"df {pod_obj.get_storage_path()} -B M --output=avail"
        )
        # Get the numeral value of available space. eg: 3070 from '3070M'
        available_size = int(df_avail_size.strip().split()[1][0:-1])
        pod_obj.run_io(
            "fs",
            size=f"{available_size-2}M",
            runtime=20,
            rate="100M",
            fio_filename=file_name,
            end_fsync=1,
        )
    log.info("Started IO on all pods to utilise 100% of PVCs")

    # Wait for IO to finish
    log.info("Wait for IO to finish on pods")
    for pod_obj in self.pods:
        pod_obj.get_fio_results()
        log.info(f"IO finished on pod {pod_obj.name}")

        # Verify used space on pod is 100%
        used_space = pod.get_used_space_on_mount_point(pod_obj)
        assert used_space == "100%", (
            f"The used space on pod {pod_obj.name} is not 100% " f"but {used_space}"
        )
        log.info(f"Verified: Used space on pod {pod_obj.name} is 100%")
        # Calculate md5sum of the file
        pod_obj.pvc.md5sum = pod.cal_md5sum(pod_obj, file_name)

    log.info("Creating clone of the PVCs")
    cloned_pvcs = [pvc_clone_factory(pvc_obj) for pvc_obj in self.pvcs]
    log.info("Created clone of the PVCs. Cloned PVCs are Bound")
    for pvc_obj in self.pvcs:
        if pvc_obj.backed_sc == constants.CEPHFILESYSTEM_SC:
            pv_obj = pvc_obj.backed_pv_obj
            subvolumname = (
                pv_obj.get()
                .get("spec")
                .get("csi")
                .get("volumeAttributes")
                .get("subvolumeName")
            )
            pend_msg = f"{subvolumname}: clone from snapshot is pending"

    # Bug 2042318
    for clone_pvc in cloned_pvcs:
        if clone_pvc.backed_sc == constants.CEPHFILESYSTEM_SC:
            pv = clone_pvc.get().get("spec").get("volumeName")
            error_msg = f"{pv} failed to create clone from subvolume"
            csi_cephfsplugin_pod_objs = res_pod.get_all_pods(
                namespace=config.ENV_DATA["cluster_namespace"],
                selector=["csi-cephfsplugin-provisioner"],
            )
        relevant_pod_logs = None
        for pod_obj in csi_cephfsplugin_pod_objs:
            pod_log = res_pod.get_pod_logs(
                pod_name=pod_obj.name, container="csi-cephfsplugin"
            )

            if pv in pod_log:
                relevant_pod_logs = pod_log
                log.info(f"Found '{pv}' on pod {pod_obj.name}")
                break

  assert (
        error_msg in relevant_pod_logs
    ), f"Logs should contain the error message '{error_msg}'"
E       TypeError: argument of type 'NoneType' is not iterable



/home/jenkins/workspace/qe-deploy-ocs-cluster-prod/ocs-ci/tests/functional/pv/pvc_clone/test_clone_when_pvc_full.py:128: TypeError
```

### Steps to Reproduce
1. Run the test case tests/functional/pv/pvc_clone/test_clone_when_pvc_full.py::TestCloneWhenFull::test_clone_when_full

### Actual Behavior
Test case failed because the provisioner pod is not identified
### Expected Behavior
The test case is expected to use the correct name and label for the ctrlplugin pod
### Impact (likelihood of reproduction, impact on the cluster and on other tests, etc.)
No impact on the cluster or other test cases
### Screenshots (if applicable)

### Environment
- Test Suite(s): tier2
- Platform(s): Any. Reported in AWS
- Version(s): 4.19 
- OS:

### Additional Context
Report portal - https://reportportal-ocs4.apps.dno.ocp-hub.prod.psi.redhat.com/ui/#ocs/launches/852/33145/1625355/1625507/log

--- a/tests/functional/pv/pvc_clone/test_clone_when_pvc_full.py
+++ b/tests/functional/pv/pvc_clone/test_clone_when_pvc_full.py
@@ -68,7 +68,7 @@ class TestCloneWhenFull(ManageTest):
             available_size = int(df_avail_size.strip().split()[1][0:-1])
             pod_obj.run_io(
                 "fs",
-                size=f"{available_size-2}M",
+                size=f"{available_size - 2}M",
                 runtime=20,
                 rate="100M",
                 fio_filename=file_name,
@@ -113,7 +113,9 @@ class TestCloneWhenFull(ManageTest):
                 error_msg = f"{pv} failed to create clone from subvolume"
                 csi_cephfsplugin_pod_objs = res_pod.get_all_pods(
                     namespace=config.ENV_DATA["cluster_namespace"],
-                    selector=["csi-cephfsplugin-provisioner"],
+                    selector=[
+                        f"{config.ENV_DATA['cluster_namespace']}.cephfs.csi.ceph.com-ctrlplugin"
+                    ],
                 )
             relevant_pod_logs = None
             for pod_obj in csi_cephfsplugin_pod_objs:

